unittests/test-contracts/kv_test/kv_test.cpp

```diff
@@ -2,33 +2,14 @@
 
 
 using namespace eosio;
+using namespace eosio::internal_use_do_not_use;
```

Comment:
Not sure if we should continue the practice of hiding the usage of the namespace `internal_use_do_not_use` with `using namespace eosio::internal_use_do_not_use;`.

Reply:
@johndebord those are unit tests that are specific to intrinsics from `internal_use_do_not_use`. code will be verbose if we will keep it. I think unit tests for intrinsics are ok for this.